File: pocketr-java.api/src/main/java/com/decrux/pocketr/api/repositories/CurrencyRepository.java
package com.decrux.pocketr.api.repositories;

import com.decrux.pocketr.api.entities.db.ledger.Currency;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface CurrencyRepository extends JpaRepository<Currency, String> {
}

File: pocketr-api/src/main/kotlin/com/decrux/pocketr/api/repositories/CurrencyRepository.kt
package com.decrux.pocketr.api.repositories

import com.decrux.pocketr.api.entities.db.ledger.Currency
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface CurrencyRepository : JpaRepository<Currency, String>
